Bottom: e60e3037bf689287ba6517cd63523e54bbd445f8
Top:    7275cec7806c80a512f25875bd90519e1f108cee
Author: Ian Kent <raven@themaw.net>
Date:   2013-02-27 14:10:33 +0800

Refresh of autofs-5.0.7-fix-systemd-unidir-in-spec-file.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 25aa5fc..50966bc 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -34,6 +34,7 @@
 - add symlink pseudo option.
 - fix requires in spec file.
 - fix libtirpc build option to require libtirpc-devel if needed.
+- fix systemd unidir in spec file.
 
 25/07/2012 autofs-5.0.7
 =======================
